tools/doc/versions.js

```diff
@@ -28,9 +38,22 @@ module.exports = {
     // The CHANGELOG.md on release branches may not reference newer semver
     // majors of Node.js so fetch and parse the version from the master branch.
     const githubContentUrl = 'https://raw.githubusercontent.com/nodejs/node/';
-    const changelog = await getUrl(`${githubContentUrl}/master/CHANGELOG.md`);
+    let changelog;
+    try {
+      changelog = await getUrl(`${githubContentUrl}/master/CHANGELOG.md`);
```

Comment:
This might be hanging with a long timeout if network is poor. Would it be possible to guard with an option to use it?

Reply:
What kind of option do you suggest?

Reply:
Read about your comment here https://github.com/nodejs/node/issues/29918#issuecomment-540932411. Would it be possible to determine if a fetch is required on if it is a release build from `node_version.h`? Since the original thought is to add a link in LTS version documents to the latest version document.

Another thing that comes up to me is that this might make release doc output unstable if the release machine has temporary internet issues.

Reply:
Or maybe we can just set a smaller timeout? IMO it's not really worth the wait when the request does not finish, say, in 5 seconds.

Reply:
I'll try working in a 5 second timeout. Note that its quite difficult for me to test this beyond yanking out my network cable (in which case the get errors very quickly). 

As much as possible I want to avoid the scenario where the release build path is only run at release time as we want to catch any potential regressions when they're introduced into the codebase rather than wait until the point we try to release.

@legendecas if the release machines have temporary internet issues then the release is at risk anyway (the machines building the releases need to clone from github.com).

Reply:
Added a 5 second timeout on the `https.get`. PTAL.